common/src/sql/dbinit.sql

```diff
@@ -916,7 +918,51 @@ CREATE TABLE omicron.public.image (
     size_bytes INT NOT NULL
 );
 
+CREATE VIEW omicron.public.project_image AS
+SELECT
+    id,
+    name,
+    description,
+    time_created,
+    time_modified,
+    time_deleted,
+    silo_id,
+    project_id,
```

Comment:
Note that project images also include `silo_id`. This is necessary to convert them back to the base image type.